src/zadatak4.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class zadatak4 {
+    public static <string> void main(String[] args) {
+
+        System.out.println("Type your word");
+        Scanner reader = new Scanner(System.in);
+        String word = reader.nextLine();
+        char[] wordChar = word.toCharArray();
+        char[] reversedWordChar = new char[wordChar.length];
+        for (int i = 0; i < wordChar.length; i++) {
+            reversedWordChar[i]=wordChar[wordChar.length-i-1];
+
+        }
+        for (int i = 0; i < wordChar.length; i++) {
+            
+          if  (reversedWordChar[i]!=wordChar[i]) {
+            System.out.println("Word is not a polindrome");
```

Comment:
but this will not test the word, but the individual chars dont give up try it again